server/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
```

Comment:
Also changing from `app.js` to `index.js` would require changing the entry point and the corresponding scripts in `package.json`